exercises/pangram/pangram_test.rb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'pangram'
+
+# Test data version:
+# 180638f Merge pull request #217 from ErikSchierboom/patch-2
+
+class PangramTest < Minitest::Test
+  # Below we use `%Q` syntax to create strings.
+  # This let's us worry less about escaping quotes.
+  # You can read more about Ruby's string literal syntax here:
+  # http://ruby-doc.org/core-2.3.0/doc/syntax/literals_rdoc.html#label-Strings
+
+  def test_sentence_empty
+    str = %Q()
+    refute Pangram.is_pangram?(str)
+  end
+
+  def test_pangram_with_only_lower_case
+    skip
+    str = %Q(the quick brown fox jumps over the lazy dog)
+    assert Pangram.is_pangram?(str)
+  end
+
+  def test_missing_character_x
+    skip
+    str = %Q(a quick movement of the enemy will jeopardize five gunboats)
+    refute Pangram.is_pangram?(str)
+  end
+
+  def test_pangram_with_mixed_case_and_punctuation
+    skip
+    str = %Q("Five quacking Zephyrs jolt my wax bed.")
```

Comment:
Here it is appropriate.  Due to the actual use of double quotes.  This will preserve the actual double quote characters inside the string and requires no escaping.  However, likely more suitable with single quoted string, as we don't require interpolation.